src/homework2/task7.py

```diff
@@ -0,0 +1,17 @@
+"""Third task for the lesson.
+
+Given: three sides of a triangle.
+Required: check if these are actually the sides of a triangle.
+If the sides define a triangle, find its area.
+If not, display an invalid data message.
+"""
+
+
+side1, side2, side3 = (int(value) for value in input('Input triangle sides separately: ').split())
+
+if (side1 < side2 + side3) and (side2 < side1 + side3) and (side3 < side1 + side2):
+    p = (side1 + side2 + side3) / 2
+    S = p * (p - side1) * (p - side2) * (p - side3) ** 0.5
```

Comment:
S - bad naming

Reply:
this formula doesn't work correctly. you can check with triangle with sides 3, 4, 5 - expected S - 6

Reply:
Changed variable name to "area" and adjusted the formula